CHANGELOG.md

```diff
@@ -60,6 +60,11 @@ yarn set version 3.0.0-rc.1
   - Projects that make use of such dependencies will have to be run with the `--preserve-symlinks` Node option if they wish to access their dependencies.
   - Because Yarn installs will never modify files outside of the project due to security reasons, sub-dependencies of packages with `portal:` must be hoisted outside of the portal. Failing that (for example if the portal package depends on something incompatible with the version hoisted via another package), the linker will produce an error and abandon the install.
 
+- The node-modules linker can now utilize hardlinks. The new setting `nmMode: classic | hardlinks | cas` specifies which `node_modules` strategy should be used:
+ - `classic` - no hardlinks will be used
+ - `hardlinks` - standard `node_modules` layout will be used with hardlinks inside the project only
+ - `cas` - global content addressable storage will be used across projects having this option
```

Comment:
It's not a problem worth blocking this PR, but I'd advise against using an acronym, and perhaps to change it in a followup PR. We don't use them a lot in the rest of the configuration, and `cas` in particular is quite cryptical (few people will understand it just by reading, and in some languages it's actually a valid word, adding to the confusion).

What about `hardlinks-global`? And perhaps rename `hardlinks` into `hardlinks-local`?

Reply:
`hardlinks-local` and `hardlinks-global` sounds good, updated